I'm getting an error in the consule when I try to use this:
```
[2017-01-23 10:38:43.676] [ERROR] console - TypeError: Object / has no method 'startsWith'
    at Object.exports.authenticate [as hook_fn] (/opt/etherpad-lite/node_modules/ep_oauth2/index.js:104:23)
    at normalizedhook (/opt/etherpad-lite/src/static/js/pluginfw/hooks.js:15:27)
    at hookCallWrapper (/opt/etherpad-lite/src/static/js/pluginfw/hooks.js:21:14)
    at /opt/etherpad-lite/src/static/js/pluginfw/hooks.js:111:7
    at next (/opt/etherpad-lite/src/static/js/pluginfw/hooks.js:46:5)
    at Object.exports.mapFirst (/opt/etherpad-lite/src/static/js/pluginfw/hooks.js:52:3)
    at Object.exports.aCallFirst (/opt/etherpad-lite/src/static/js/pluginfw/hooks.js:108:11)
    at authenticate (/opt/etherpad-lite/src/node/hooks/express/webaccess.js:47:11)
    at /opt/etherpad-lite/src/node/hooks/express/webaccess.js:83:5
    at /opt/etherpad-lite/src/node/hooks/express/webaccess.js:15:14
```

Sorry I can't be of more help but I've zero experience with these kinds of apps, am I missing a dependency perhaps? I'm using a docker container with Debian Jessie, is the default nodejs there too old?

--- a/index.js
+++ b/index.js
@@ -101,7 +101,7 @@ exports.expressCreateServer = function (hook_name, context) {
 }
 
 exports.authenticate = function(hook_name, context) {
-  if (context.req.url.startsWith('/auth/')) return context.next();
+  if (context.req.url.indexOf('/auth/') === 0) return context.next();
   console.info('oauth2-authenticate from ->', context.req.url);
   context.req.session.afterAuthUrl = context.req.url;
   return passport.authenticate('hbp')(context.req, context.res, context.next);

